src/tcgwars/logic/impl/pokemod/PokemodFossil.groovy

```diff
@@ -130,129 +182,871 @@ public enum PokemodFossil implements LogicCardInfo {
   public Card getImplementation() {
     switch (this) {
       case AERODACTYL_1:
-        break
+        return evolution (this, from:"Mysterious Fossil", hp:HP060, type:FIGHTING, retreatCost:2) {
+          weakness GRASS
+          resistance FIGHTING, MINUS30
+          pokemonPower "Prehistoric Power", {
+            text "No more Evolution cards can be played. This power stops working while Aerodactyl is Asleep, Confused, or Paralyzed."
+            delayedA {
+              if (self.cards.filterByType(POKEMON_TOOL)) {
+              before PLAY_EVOLUTION, {
+                if(!self.specialConditions){
+                  wcu "Prehistoric Power prevents you from playing Evolution cards!"
+                  prevent()
+                }
+              }
+            }
```

Comment:
```suggestion
              before PLAY_EVOLUTION, {
                if(!self.specialConditions && self.cards.filterByType(POKEMON_TOOL)){
                  wcu "Prehistoric Power prevents you from playing Evolution cards!"
                  prevent()
                }
              }
            }
```
You need to check the tool is attached inside of the `before`. Checking it outside will prevent the delayed effect from ever being registered if you don't have a tool attached when Aerodactly comes into play, even if you attach it later.